pkg/operation/common/utils.go

```diff
@@ -223,7 +223,7 @@ func GetLoadBalancerIngress(client kubernetes.Interface, namespace, name string)
 	if serviceStatusIngress[length-1].IP != "" {
 		loadBalancerIngress = serviceStatusIngress[length-1].IP
 	} else if serviceStatusIngress[length-1].Hostname != "" {
-		loadBalancerIngress = serviceStatusIngress[length-1].Hostname
+		loadBalancerIngress = fmt.Sprintf("%s.",serviceStatusIngress[length-1].Hostname)
```

Comment:
Not sure that we should do it here. We don't know whether what has been put into the `hostname` field is a FQDN.
Instead, I suggest fixing it in the aws-lb-readvertiser itself. Here, the contract is that the load balancer name is passed via command line flag - and if it's no FQDN then the aws-lb-readvertiser can convert it to a FQDN.
OK?

Reply:
Ok